.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+repos:
+  - repo: local
+    hooks:
+      - id: ruff
+        name: Ruff
+        entry: ruff check --fix
+        language: system
+        types: [ python ]
+
+      - id: isort
```

Comment:
@vasylboliuk please remove isort and flake8 checks. Ruff contains it already. https://docs.astral.sh/ruff/
please also remove it from project dependencies in pyproject.toml

instead pleas add another hook for `ruff format` command prior to `ruff check` hook

Reply:
Done